test/util/test_runners/integration_test_case.py

```diff
@@ -0,0 +1,71 @@
+"""
+extend the unitest TestCase class
+"""
+
+# Builtin
+import io
+import sys
+
+# 3rp parties
+import itertools
+from unittest import TestCase
+from contextlib import redirect_stdout
+import subprocess as sp
+from unittest.mock import patch, MagicMock
+
+# internals
+
+from conjur.cli import Cli
+from test.util.test_runners.params import ClientParams, TestEnvironmentParams
+
+
+class IntegratioTestCaseBase(TestCase):
+
+    def __init__(self, testname, client_params: ClientParams = None,
```

Comment:
can you provide an example of how you run the two flows in the PR description?

Reply:
will add